scripts/superdesk/datetime/datetime.js

```diff
@@ -150,11 +150,12 @@ export default angular.module('superdesk.datetime', [
 
     .directive('sdDatetime', DateTimeDirective)
 
-    .filter('reldate', function reldateFactory() {
+    .filter('reldate', ['gettextCatalog', function reldateFactory(gettextCatalog) {
         return function reldate(date) {
+            moment.locale(gettextCatalog.currentLanguage);
```

Comment:
I'm not an expert with moment, but should this not be a project-wide setting? Does the locale only apply to this filter? Won't changing the locale here change it everywhere?

If so, maybe this would be best suited in `scripts/bootstrap.js` inside a `run` clause, such as:
```js
.run(['gettextCatalog', gtc => moment.locale(gtc.currentLanguage)]);
```